iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -39,11 +39,6 @@ final class MainViewController: UIViewController, CardViewControllerDelegate, Th
         cardViewController?.addressViewController.setCountrySelected(country: Self.countryGB)
     }
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        UIFont.loadAllCheckoutFonts
```

Comment:
project is using same fonts for Frames, please undelete it

Reply:
✅ I think I misunderstood what you meant [here](https://checkout.slack.com/archives/GS5KC6HU3/p1659452713661439?thread_ts=1659452584.733489&cid=GS5KC6HU3) then. It appeared to me font is registered by SDK so no more need for this.